src/functions-and-arrays.js

```diff
@@ -1,41 +1,92 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
-
+function maxOfTwoNumbers(a, b) {
+  if (a > b) {
+    return a;
+  }
+  else {
+    return b
+  }
+}
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
+function findLongestWord(words) {
+  let maxLetters = words[0];
+  for (let i = 0; i < words.length - 1; i++) {
+    if (words[i].length >= maxLetters.length) {
+      maxLetters = words[i]
+    }
```

Comment:
Ese menos 1 te está causando que no recorras el array hasta el final, te dejas fuera siempre el último elemento.
```suggestion
  for (let i = 0; i < words.length; i++) {
    if (words[i].length >= maxLetters.length) {
      maxLetters = words[i]
    }
```

Además para retornar la primera no has de sustituirla cuando tengan la misma longitud:
```suggestion
  for (let i = 0; i < words.length - 1; i++) {
    if (words[i].length > maxLetters.length) {
      maxLetters = words[i]
    }
```